packages/remirror__extension-list/src/list-commands.ts

```diff
@@ -151,20 +147,29 @@ export function splitListItem(listItemType: string | NodeType): CommandFunction
     }
 
     const nextType = $to.pos === $from.end() ? grandParent.contentMatchAt(0).defaultType : null;
-    tr.delete($from.pos, $to.pos);
 
-    const types: TypesAfter = [undefined];
+    const newTypeAttributes = Object.fromEntries(
+      Object.entries(grandParent.attrs).filter(([attr]) => !ignoreAttrs.includes(attr)),
+    );
 
-    if (nextType) {
-      types.push({ type: nextType, attrs: {} });
-    }
+    const newNextTypeAttributes = { ...$from.node().attrs };
```

Comment:
Can we rename this to `contentAttributes`? Since it is actually the content inside the item being split.

```suggestion
    const contentAttributes = { ...$from.node().attrs };
```

Reply:
@ifiokjr Done. I also rename some other variables in this function (`listItemType`, `listItemAttributes`, `contentType`, `contentAttributes`)

Reply:
Nice!